<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParamImages</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLoadImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить изображение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIncrease">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Увеличить масштаб (+)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecrease">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Уменьшить масштаб (-)</string>
    </property>
   </widget>
   <widget class="MyQGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>631</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveRezults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить параметры</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры</string>
    </property>
    <widget class="QTextEdit" name="txtParamValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Клетки из файла контуров</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="showSquare">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить клетку</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="showContour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить контур</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет выбранного объекта</string>
    </property>
    <widget class="QLabel" name="selColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelColor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>26</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим полупрозрачности</string>
    </property>
    <widget class="QCheckBox" name="isAlpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Включен</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Полупрозрачность</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinAlpha">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelColor_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>116</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QLabel" name="selColor_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбрать сортируемый параметр</string>
    </property>
    <widget class="QRadioButton" name="radioB1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Средняя яркость R</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Средняя яркость G</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Средняя яркость B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>СКО R</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>СКО G</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>СКО B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Диапазон R</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Диапазон G</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Диапазон B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Текстур призн МПС R</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Текстур призн МПС G</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Текстур призн МПС B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Тек пр длины серий R</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Тек пр длины серий G</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Тек пр длины серий B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Форма</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Площадь</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Периметр</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioB0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>По номеру</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Сортировка</string>
    </property>
    <widget class="QRadioButton" name="ascBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>По возрастанию</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="descBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>По убыванию</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_asc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_desc">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">myqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
